devel/libgit2/Portfile

```diff
@@ -7,7 +7,7 @@ PortGroup           legacysupport 1.0
 PortGroup           muniversal 1.0
 
 # don't forget to update py-pygit2 and libgit2-glib as well
-github.setup        libgit2 libgit2 1.3.0 v
+github.setup        libgit2 libgit2 1.4.2 v
 github.tarball_from archive
 revision            1
```

Comment:
the revision field should be reset to zero when updating the version

Reply:
@reneeotten Thank you. Fixed.